cirq/google/engine/calibration.py

```diff
@@ -50,9 +50,15 @@ class Calibration(abc.Mapping):
             the epoch.
     """
 
-    def __init__(self, calibration: v2.metrics_pb2.MetricsSnapshot) -> None:
+    def __init__(self,
+                 calibration: v2.metrics_pb2.MetricsSnapshot = v2.metrics_pb2.
+                 MetricsSnapshot(),
```

Comment:
this is a weird line wrap. cc: that bug about how we need a better formatter

Reply:
Fixed formatting with parentheses.

Reply:
cc #3292